src/systems/odometry_publisher/OdometryPublisher.cc

```diff
@@ -158,6 +165,11 @@ void OdometryPublisher::Configure(const Entity &_entity,
         "rpy_offset"));
   }
 
+  if (_sdf->HasElement("gaussian_noise"))
+  {
+    this->dataPtr->gaussianNoise = _sdf->Get<double>("gaussian_noise");
```

Comment:
Should the new sdf elements (`gaussian_noise` and `odom_covariance_topic`) be documented somewhere?

Reply:
Done